### DISCUSSION TITLE
Add option to download all attachments in a form's gallery view

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5848] [Feature Requests]
Add a button to download all images in an attachment gallery view in a form. 

![Screenshot 2023-07-19 at 10 42 41 AM](https://github.com/specify/specify7/assets/135047322/ee3bf062-607f-473e-8817-105ab10c0a9f)

![Screenshot 2023-07-19 at 10 40 18 AM](https://github.com/specify/specify7/assets/135047322/c05690c5-02ea-4b44-b14a-d402d69d122d)

**Reported By**
Grant (AAFC-AAC requested this functionality)

**Comment by maxpatiiuk** [comment]
not related, but the positioning of the attachments icon is weird (see 1st screenshot) - it should either be to the left of the slider, next to the +/- icons, or should be styled to look like a button
